presto-hive/src/main/java/com/facebook/presto/hive/HiveBucketing.java

```diff
@@ -126,7 +126,7 @@ private static int hash(TypeInfo type, Block block, int position)
                         long bigintValue = prestoType.getLong(block, position);
                         return (int) ((bigintValue >>> 32) ^ bigintValue);
                     case FLOAT:
```

Comment:
Did you check on that list of types at the top of this file? If you aren't going to address it now, please file a ticket so we don't forget